### PR TITLE
Create tab navigator

### PR DESCRIPTION
**Pull request by Chrischuck**
#499 & #462

https://user-images.githubusercontent.com/13311268/221372286-f8920da1-0c9c-4428-8844-097254f5c23a.mov

### CODE REVIEW

File: src/components/ObsDetails/ObsDetails.js [hunk 1]
```diff
@@ -202,8 +203,16 @@ const ObsDetails = ( ): Node => {
...
+      <HeaderBackButton
+        tintColor={colors.black}
+        onPress={navigation.goBack}
+      />
+    )
+
     navigation.setOptions( {
-      headerRight: editIcon
+      headerRight: editIcon,
+      headerLeft: backButton
```
Comment (Chrischuck) [inline src/components/ObsDetails/ObsDetails.js:215]: I'm not sure how this page had a back button w/o this code previously. But for what it's worth, I think that this should be defined w/ the screen definition in the navigator.

File: src/realmModels/User.js [hunk 2]
```diff
@@ -23,6 +23,8 @@ class User extends Realm.Object {
 
   static currentUser = realm => realm.objects( "User" ).filtered( "signedIn == true" )[0]
 
+  static firstSignedInUser = realmObjects => realmObjects.filtered( "signedIn == true" )[0]
```
Comment (jtklein) [inline src/realmModels/User.js:26]: This deos not seem to be used somewhere.

**Review by jtklein — CHANGES_REQUESTED** [review]
Hi @Chrischuck, have now managed to look through all of this PR. For the most part it looks good to me. Some issues I have seen in the apps:

On Android the back button is not visible, you can tap it though. I haven't checked if it is transparent or same color as the background, or a text with no string or ...

There is no more back button in the Taxon details screen.

Now that the footer is no longer there, these two pairs of components can be combined and renamed, as they are identical now:
ScrollNoFooter & ScrollWithFooter
ViewNoFooter & ViewWithFooter

File: src/components/ObsDetails/ObsDetails.js (hunk 1 above)
Comment (Chrischuck) [reply src/components/ObsDetails/ObsDetails.js:215]: Can do, for future reference, in a Stack navigator, the back button gets automatically added if it is possible to go back. But when we switched to tab navigation, the behavior did not follow

**Pull request merged**